game-core/src/main/java/games/strategy/engine/framework/ui/GameChooser.java

```diff
@@ -66,7 +66,7 @@ private void layoutComponents() {
     leftPanel.setLayout(new GridBagLayout());
     final JLabel gamesLabel = new JLabel("Games");
     gamesLabel.setFont(
-        gamesLabel.getFont().deriveFont(Font.BOLD, gamesLabel.getFont().getSize() + 2));
+        gamesLabel.getFont().deriveFont(Font.BOLD, gamesLabel.getFont().getSize() + 2.0F));
```

Comment:
```suggestion
        gamesLabel.getFont().deriveFont(Font.BOLD, gamesLabel.getFont().getSize() + 2f));
```
I don't think we use `F` for floats, just `f`?
In any case the trailing `.0` is not required

Reply:
We have more `f` than uppercase `F`, but it is mixed.
The `.0` I like as `1f` and `2f` look like hex numbers.